src/bootstrap/src/core/config/flags.rs

```diff
@@ -32,6 +32,19 @@ pub enum Warnings {
     Default,
 }
 
+#[derive(Copy, Clone, Debug, ValueEnum, PartialEq, Eq)]
+pub enum FerroceneCoverageFor {
```

Comment:
Should this be non_exhaustive?

```suggestion
#[non_exhaustive]
pub enum FerroceneCoverageFor {
```

Reply:
No, we want it to break compilation if we add a new variant.